pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+| Solution                            |                                            |                           |
+| SolutionComponent                   |                                            |                           |
+| UrlValue                            |                                            | AttributeValue            |
+| WorkflowExecutionActivity           |                                            |                           |
+|                                     | CalibrationInformation                     |                           |
+|                                     | ChemicalConversionProcess                  |                           |
+|                                     | ChromatographyConfiguration                |                           |
+|                                     | Configuration                              |                           |
+|                                     | DissolvingProcess                          |                           |
+|                                     | InformationObject                          |                           |
+|                                     | Instrument                                 |                           |
+|                                     | MassSpectrometry                           |                           |
+|                                     | MassSpectrometryConfiguration              |                           |
+|                                     | MaterialProcessing                         |                           |
+|                                     | MobilePhaseSegment                         |                           |
+|                                     | NucleotideSequencing                       |                           |
+|                                     | PortionOfSubstance                         |                           |
+|                                     | ProtocolExecution                          |                           |
+|                                     | StorageProcess                             |                           |
+|                                     | WorkflowExecution                          |                           |
+
+For example, `MetagenomeAnnotationActivity` used to be located in the hierarchy
+`Activity.WorkFlowExecutionActivity`, but now it and 11 similar classes are located in
+`NamedThing.PlannedProcess.WorkflowExecution`. The word `Activity`, inherited from the Provenance Ontology, has largely
+been removed from the schema, as `PlannedProcess` from the Ontology of Biomedical Investigations now plays a more
+important organizational role.
+
+## v10.9.1
+
+![Activity.png](Activity.png)
+
+`ProtocolExecution` has been added as a `PlannedProcess` for aggregating other processes together when they follow the
+same `Protocol`.
```

Comment:
At this point, I found myself wondering "are these formatted words all names of classes?"

Reply:
in any documentation I write, backticks indicate the verbatim name of an element in the schema or a variable, function, etc in Python code

Reply:
Added comment to Conventions section at top of document

Reply:
OK. Even with that info, I am wondering whether these are names of _classes_ (verbatim from the schema) or names of _some other kind of thing_ (verbatim from the schema).

Reply:
I have added more section headers, more conventions definitions, and more careful use of backticks